lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -563,6 +563,14 @@
     <datatype extension="gal" type="galaxy.datatypes.microarrays:Gal" display_in_upload="true"/>
     <datatype extension="rdata" type="galaxy.datatypes.binary:RData" display_in_upload="true" description="Stored data from an R session"/>
     <datatype extension="rdata.sce" type="galaxy.datatypes.binary:RData" description="Stored RDS from a SingleCellObject" subclass="true" display_in_upload="true"/>
+    <datatype extension="rdata.msnbase.raw" type="galaxy.datatypes.binary:RData" mimetype="application/x-gzip" subclass="true" display_in_upload="true" />
```

Comment:
is the mimetype correct and do we need it to other rdata-types?

Reply:
I think it is - I used `file --mime-type -b filename.rdata` to get the mime type of the rdata files produced by XCMS and CAMERA tools.

Reply:
Good point. Since the other data types derived from Rdata also don't have  a mime type. We should have this consistently.

Reply:
... or remove it.

Reply:
I have added them to the other two Rdata data types for consistency.